EIPS/eip-2502.md

```diff
@@ -0,0 +1,462 @@
+---
+eip: 2502
+title: µTopic Delegation
+author: Ricardo Guilherme Schmidt (@3esmit)
+discussions-to: https://ethereum-magicians.org/t/eip-2502-topic-delegation/3974
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-02-02
+requires: 2470
+
+---
+
+## Simple Summary
+
+A common trust network for ethereum.
+
+A general purpose hierarchy of delegations which can divided into micro topics. 
```

Comment:
```suggestion
A general purpose hierarchy of delegations which can divided into micro topics.
```
Simple Summary should be akin to an email subject line, forum thread title, GitHub Pull Request title line, etc.